### ISSUE TITLE
Deploy: Create a ZOOM IBEX Server/Client System with MDT

### ISSUE THREAD

**Issue by ChrisM-S** [ISISComputingGroup/IBEX#2490]
As a systems maintainer I would like to be able to upgrade and re-build instrument control systems automatically in a short time and to a manifest of basic platform utilities. This will be demonstrated on ZOOM.

The following tickets must be done first:

 - [ ] #2492
 - [ ] #2426 
 - [ ] A data VHD (I think this is done hence no ticket)
 - [ ] #2493

**Acceptance Criteria**

**Notes**
1. The System will build automatically
2. The IBEXTEST system will the starting point
3. Sort out which functions are most critical

**Comment by John-Holt-Tessella** [comment]
Duplicate of #2428
